lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeNames.cpp

```diff
@@ -419,6 +429,66 @@ CreateRunThroughTaskSwitchingTrampolines(Thread &thread,
   return nullptr;
 }
 
+// Search all modules for `target_func` and creates a RunToAddress plan if a
+// single function is found.
+static ThreadPlanSP CreateRunToAddressPlan(StringRef target_func,
+                                           Thread &thread, bool stop_others) {
+  ModuleList modules = thread.GetProcess()->GetTarget().GetImages();
+  SymbolContextList sc_list;
+  modules.FindFunctionSymbols(ConstString(target_func), eFunctionNameTypeFull,
+                              sc_list);
+  if (sc_list.GetSize() != 1 || sc_list[0].symbol == nullptr)
+    return nullptr;
```

Comment:
```suggestion
    return {};
```

Reply:
I'm a bit confused by this suggestion, the function returns a pointer

Reply:
It returns a ThreadPlanSP (the success return returns make_shared).   Apparently C++ is smart enough to turn nullptr into a default constructed shared pointer, but I agree with Adrian, that's a little magical for my taste.